vtex/faststore
# Chore: Carousel items per page from cms

## What's the purpose of this pull request?

Adds `itemsPerPage` field for Carousel in the ProductShelf section. 

## How it works?

You should be able to set the carousel's itemsPerPage prop via CMS.

## How to test it?

- in faststore.config.js change the workspace to fanny
- `yarn dev` to run locally
- check the ProductShelf in homepage using these links (version from fanny's workspace):

http://localhost:3000/api/preview?contentType=home&documentId=509f861b-719d-11ee-83ab-0a650ce03a3d&versionId=52f602cb-719d-11ee-83ab-126d1b33ed09

Check in the CMS:
- https://fanny--storeframework.myvtex.com/admin/headless-cms/faststore -> Home -> ProductShelf section

You should be able to see the `itemsPerPage` field

<img width="798" alt="image" src="https://github.com/vtex/faststore/assets/3356699/9d7a6398-d8d3-4010-bf1d-c5ee952e98fd">


### Starters Deploy Preview
https://github.com/vtex-sites/starter.store/pull/291
